android/sprint_process.md

```diff
@@ -0,0 +1,62 @@
+# APT sprint process
+Projects created by the Android Product Team typically ship a major release every six weeks and ship a minor release twice between major releases (every two weeks). For example, if v2.0 is released on week 0, v2.1 will be released on week 2, v2.2 will be released on week 4, and v3.0 will be released on week 6. Emergency bug fixes can be released more frequently. Projects will typically link to their release schedule (example: [focus-android](https://wiki.mozilla.org/Mobile/Focus/Android/Train_Schedule)).
+
+Development is broken down into two week-long chunks of time called sprints: we track the work for these sprints in GitHub milestones (example: [focus-android][milestone example]). We have several planning meetings to identify what should go into each release and thus into each sprint.
+
+## Sprint planning meetings
+### Roadmap (monthly)
+The product team plans and prioritizes the project roadmap: the general project plan over the next few months. These meetings will:
+- At a high level, decide which features go into which releases
+- Schedule team work weeks
+- Modify the release schedule (if necessary)
+
+Updates from this meeting will be emailed to the full team.
+
+### Triage (weekly)
+The full team will go through newly filed issues to:
+- Assign priority labels (P1, P2, ...) to bugs without priority labels or the `addressed` label
+- Ensure all P1 labels are assigned to a milestone
+- Ensure all P2 labels are assigned to a milestone
+
+Each project should link to its own triage queries (example: [focus-android](https://github.com/mozilla-mobile/focus-android/issues?q=is%3Aissue+is%3Aopen+-label%3AP1+-label%3AP2+-label%3AP3+-label%3AP4+-label%3AP5+-label%3Aaddressed+-label%3Ablocked+sort%3Aupdated-desc+no%3Amilestone)).
+
+For more information on priority labels, [see below](#categorizing-issues).
+
+### Bug management (weekly)
```

Comment:
This might just be me, but reading this for the first time I thought "Bug management" was a meeting for managing issues labeled as `bug`. I'm not sure I like "Issue management" better though 🙃

Reply:
Changed to "Weekly bug planning"

Reply:
That makes the tag redundant but whatever :)